### PR TITLE
Feature/prepare demo app for common properties

### PR DESCRIPTION
**Pull request by achocron**
For now, just review Sasquatch app changes - puppet is not yet ready

### CODE REVIEW

File: TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import UIKit
 
-private var kPropertiesSection: Int = 3
+private var kEventPropertiesSection: Int = 2
+private var kTargetPropertiesSection: Int = 3
```
Comment (jwallra) [inline TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift]: This is a section _index_, no?  Just a naming nit :)

File: TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift [hunk 2]
```diff
@@ -29,127 +34,96 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
...
+    return super.tableView(tableView, heightForRowAt: indexPath)
   }
 
   override func tableView(_ tableView: UITableView, indentationLevelForRowAt indexPath: IndexPath) -> Int {
-    if isPropertiesRowSection(indexPath.section) {
-      return super.tableView(tableView, indentationLevelForRowAt: IndexPath(row: 0, section: indexPath.section))
-    } else {
-      return super.tableView(tableView, indentationLevelForRowAt: indexPath)
-    }
+    return 0
```
Comment (jwallra) [inline TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift]: What is this function doing now?  Is it still needed?

Comment (achocron) [reply TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift]: It is still needed - there's a crash if it's removed. I can add a comment.

File: TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift (hunk 1 above)
Comment (achocron) [reply TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift]: Sure, will update

File: TestApps/Sasquatch/Sasquatch/ViewControllers/PropertiesTableSection/TargetPropertiesTableSection.swift [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  override func numberOfCustomHeaderCells() -> Int {
+    return 1
+  }
+
+  override func propertyKeyChanged(sender: UITextField!) {
+    let selectedTarget = transmissionTargetSelectorCell?.selectedTransmissionTarget()
+    let arrayIndex = getCellRow(forTextField: sender) - propertyCellOffset()
+    let currentPropertyKey = targetProperties[selectedTarget!]![arrayIndex].0
+    let currentPropertyValue = targetProperties[selectedTarget!]![arrayIndex].1
+    let target = MSAnalytics.transmissionTarget(forToken: selectedTarget!)
```
Comment (clpolet) [inline TestApps/Sasquatch/Sasquatch/ViewControllers/PropertiesTableSection/TargetPropertiesTableSection.swift]: You should get the target from the list of target here I think :);

File: TestApps/Sasquatch/Sasquatch/ViewControllers/PropertiesTableSection/TargetPropertiesTableSection.swift [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    target.removeEventPropertyforKey(currentPropertyKey)
+    target.setEventPropertyString(currentPropertyValue, forKey: sender.text!)
+    targetProperties[selectedTarget!]![arrayIndex].0 = sender.text!
+  }
+
+  override func propertyValueChanged(sender: UITextField!) {
+    let selectedTarget = transmissionTargetSelectorCell?.selectedTransmissionTarget()
+    let arrayIndex = getCellRow(forTextField: sender) - propertyCellOffset()
+    let currentPropertyKey = targetProperties[selectedTarget!]![arrayIndex].0
+    let target = MSAnalytics.transmissionTarget(forToken: selectedTarget!)
```
Comment (clpolet) [inline TestApps/Sasquatch/Sasquatch/ViewControllers/PropertiesTableSection/TargetPropertiesTableSection.swift]: You should get the target from the list of target here I think :);

**Pull request merged**